R/readIEA_WEIO_2014.R

```diff
@@ -1,26 +1,131 @@
-#' @importFrom readxl read_xls
+#' IEA World Energy Investment Outlook (2014)
+#'
+#' Read projected 2014-20 investments into industry energy efficiency from the
+#' [IEA World Energy Investment Outlook (2014)](http://www.iea.org/publications/freepublications/publication/weo-2014-special-report---investment.html)
```

Comment:
will this cause problems if we want to distribute REMIND input data?

Reply:
The report is available on the IEA website: https://www.iea.org/reports/world-energy-investment-outlook  I can't speak to the terms under which it can be used.